pkg/config/config.go

```diff
@@ -29,9 +29,14 @@ type Config struct {
 }
 
 type IgnoreEntry struct {
-	ID          string    `toml:"id"`
-	IgnoreUntil time.Time `toml:"ignoreUntil"`
-	Reason      string    `toml:"reason"`
+	ID             string    `toml:"id"`
+	IgnoreUntil    time.Time `toml:"ignoreUntil"`
+	Reason         string    `toml:"reason"`
+	IncludeAliases *bool     `toml:"includeAliases"` // Use pointer to tell apart unset (treated as true) and false
```

Comment:
Maybe change the name to ExcludeAliases, so defaulting to false is automatic, and makes it clearer that default is to include. (Not 100% sure about this, IncludeAliases also makes sense to me)

Reply:
Removed the flag entirely